<!doctype html>
<html lang="en">
<head>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

    <!-- Bootstrap CSS -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css" integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">

    <!-- Custom CSS -->
    <link rel="stylesheet" href="style.css">

    <title>Shea Group &SmallCircle; People</title>
</head>
<body>


    <header>
        <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
          <a class="navbar-brand" href="#">Shea Group</a>
          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
          </button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav mr-auto">
              <li class="nav-item">
                <a class="nav-link" href="./index.html">Home <span class="sr-only">(current)</span></a>
              </li>
              <li class="nav-item active">
                <a class="nav-link" href="./people.html">People</a>
              </li>
              <li class="nav-item">
                <a class="nav-link" href="./research.html">Research</a>
              </li>
              <li class="nav-item">
                <a class="nav-link" href="./publication.html">Publications</a>
              </li>
            </ul>
          </div>
        </nav>
    </header>

    <br>
    <br>

    <h3 id="role-title-pi" class="role-title">Principal Investigator</h3>

    <div id="accordion-pi">
        <div class="card">
            <div class="card-header" id="Shea">
            <h5 class="mb-0 minor-title-head">
                <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#pi-collapseOne" aria-expanded="false" aria-controls="pi-collapseOne">
                Joan-Emma Shea &SmallCircle; Physical Chemistry, Theory/Computation &amp; Biochemistry/Biophysics
                </button>
            </h5>
            </div>
            <div id="pi-collapseOne" class="collapse" aria-labelledby="headingOne" data-parent="#accordion-pi">
            <div class="mb-0 card-body">
                <div>
                    <p style="float: left; padding-right: 2rem; margin-bottom: 15rem"><img class="people-profile-pic" src="Images/People/Shea/shea2.jpg" alt="A picture of Joan-Emma Shea"></p>
                    </div>
                    <div>
                    <h1 class="research-header">Research</h1>
                    <p class='profile-statement'> Research in the Shea group focuses on developing and applying the techniques of statistical and computational physics to the study of biological problems. Current work involves the investigation of cellular processes such as in-vivo protein folding and protein aggregation. </p>
                    <h1 class="publication-header">Featured Publications</h1>
                    <a href="https://doi.org/10.1021/ja3056694" target="_blank">
                        <strong>Computational and Experimental Analyses Reveal the Essential Roles of Interdomain Linkers in the Biological Function of Chemotaxis Histidine Kinase CheA</strong>
                        </a>
                        <br />
                        <p>Wang, X.; Wu, C.; Vu, A.; Shea, J.-E.; Dahlquist, F. W. <em>J. Am. Chem. Soc.</em> <strong>2012,</strong> <em>134 (39), 16107-16110.</em></p>

                        <a href="https://doi.org/10.1016/j.jmb.2012.09.012" target="_blank">
                            <strong>The Right Angle (RA) Motif: A Prevalent Ribosomal RNA Structural Pattern Found in Group I Introns</strong>
                            </a>
                            <br />
                            <p>Grabow, W. W.; Zhuang, Z.; Swank, Z. N.; Shea, J.-E.; Jaeger, L. <em>J. Mol. Biol.</em> <strong>2012,</strong> <em>424</em> 54-67.</p>
                    <h1 class="contact-header">Contact Info</h1>
                    <p> shea@chem.ucsb.edu &SmallCircle; <a href='https://www.chem.ucsb.edu/people/joan-emma-shea' target="_blank"> Department of Chemistry and Biochemistry</a></p>
                </div>
            </div>
        </div>
    </div>

    <h3 id="role-title-postdoc" class="role-title">Postdoctoral Fellows</h3>

    <div id="accordion-postdoc">
        <div class="card">
            <div class="card-header" id="Cunhas">
            <h5 class="mb-0 minor-title-head">
                <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#pd-collapseOne" aria-expanded="false" aria-controls="pd-collapseOne">
                Keila Cunha &SmallCircle; Force Field Parameterization, Protein-Surface Adhesion &amp; Molecular Dynamics Simulation
                </button>
            </h5>
            </div>
            <div id="pd-collapseOne" class="collapse" aria-labelledby="headingOne" data-parent="#accordion-postdoc">
            <div class="mb-0 card-body">
                <div>
                    <p style="float: left; padding-right: 2rem; margin-bottom: 15rem"><img class="people-profile-pic" src="Images/People/keila.jpg" alt="A picture of Keila Cunhas"></p>
                    </div>
                    <div>
                    <h1 class="research-header">Research</h1>
                    <p class='profile-statement'> My work uses molecular dynamics to study how mussels use specialized foot proteins (mfps) to attach to mineral surfaces.
                        I also am developing force fields to produce better models of N-substituted glycine oligomers, peptoids, which are of interest due to their protease resistance and high stability compared to peptides. </p>
   
                    <h1 class="publication-header">Featured Publications</h1>
                    <a href="https://doi.org/10.1021/acs.macromol.9b02715" target="_blank">
                        <strong>Dueling backbones: Comparing peptoid and peptide analogues of a mussel adhesive protein</strong>
                        </a>
                        <br />
                        <p>Wonderly, W. R.; Cristiani, T. R.; Cunha, K. C.; Degen, G. D.; Shea, J.-E.; Waite, J. H. <em>Macromolecules</em> <strong>2020,</strong> <em>53 (16), 6767-6779.</em></p>

                        <a href="http://dx.doi.org/10.1002/adma.201703026" target="_blank">
                            <strong>Significant performance enhancement of polymer resins by bioinspired dynamic bonding</strong>
                            </a>
                            <br />
                            <p>Seo, S.; Lee, D. W.; Ahn, J. S.; Cunha, K.; Ju, S. W.; Shin, E.; Kim, B.-S.; Levine, Z. A.; Lins, R. D.; Israelachvili, J. N.; Waite, J. H.; Shea, J.-E.; Ahn, B. K. <em>Adv. Mater.</em> <strong>2017,</strong> <em>29</em> 1703026.</p>
                    <h1 class="contact-header">Contact Info</h1>
                    <p> kcunha@chem.ucsb.edu &SmallCircle; <a href='https://www.linkedin.com/in/keila-cunha-1a5454133/' target="_blank"> LinkedIn</a></p>
                </div>
            </div>
        </div>
        <div class="card">
            <div class="card-header" id="Ganguly">
            <h5 class="mb-0 minor-title-head">
                <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#pd-collapseTwo" aria-expanded="false" aria-controls="pd-collapseTwo">
                Pritam Ganguly &SmallCircle; Cosolvent Effects on Proteins, Thermodynamic Theory &amp; Atomistic Simulation
                </button>
            </h5>
            </div>
            <div id="pd-collapseTwo" class="collapse" aria-labelledby="headingTwo" data-parent="#accordion-postdoc">
            <div class="card-body">
                <div>
                    <p style="float: left; padding-right: 2rem; margin-bottom: 15rem"><img class="people-profile-pic" src="Images/People/pritam.jpg" alt="A picture of Pritam Ganguly"></p>
                    </div>
                    <div>
                    <h1 class="research-header">Research</h1>
                    <p class='profile-statement'>I study the effects of cosolvents on proteins. We are interested in understanding the effects of cellular components
                        on the conformational stability and aggregation propensities of intrisically disordered proteins (IDPs). I have particular interest in the behavior
                    of the IDP tau protein, associated with the pathology of Alzheimer's disease.</p>
   
                    <h1 class="publication-header">Featured Publications</h1>
                    <a href="https://doi.org/10.1021/jacs.0c12729" target="_blank">
                        <strong>Catalytic Cross Talk Between Key Peptide Fragments that Couple Alzheimer's Disease with Amyotrophic Lateral Sclerosis</strong></a>
                        <br />
                         <p>Laos, V.; Bishop, D., Ganguly; Schonfeld, G.; Trapp, B.; Lazar Cantrell, K.; Buratto, S.; Shea, J.-E.; Bowers, M. <em>J. Am. Chem. Soc.</em> <strong>2021,</strong> <em>143 (9), 3494-3502</em></p>

                         <a href="https://doi.org/10.1021/acs.jpcb.0c05768" target="_blank">
                            <strong>Terminal capping of an amyloidogenic Tau fragment modulates its fibrillation propensity</strong>
                            </a>
                            <br />
                            <p>Arya, S.; Ganguly, P.; Arsiccio, A.; Claud, S. L.; Trapp, B.; Schonfeld, G. E.; Liu, X.; Lazar Cantrell, K. L.; Shea, J.-E.; Bowers, M. T. <em>J. Phys. Chem. B</em> <strong>2020,</strong> <em> 124 (40), 8772-8783.</em></p>
                    <h1 class="contact-header">Contact Info</h1>
                    <p> pganguly@chem.ucsb.edu &SmallCircle; <a href='http://people.chem.ucsb.edu/ganguly/pritam' target="_blank"> Website</a></p>
                </div>
            </div>
            </div>
        </div>
        <div class="card">
            <div class="card-header" id="Najafi">
            <h5 class="mb-0 minor-title-head">
                <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#pd-collapseThree" aria-expanded="false" aria-controls="pd-collapseThree">
                Saeed Najafi &SmallCircle; Statistical Analysis, Biophysics &amp; Clathrate Nucleation
                </button>
            </h5>
            </div>
            <div id="pd-collapseThree" class="collapse" aria-labelledby="headingThree" data-parent="#accordion-postdoc">
            <div class="card-body">
                <div>
                    <p style="float: left; padding-right: 2rem; margin-bottom: 15rem"><img class="people-profile-pic" src="Images/People/saeed.jpg" alt="A picture of Saeed Najafi"></p>
                    </div>
                    <div>
                    <h1 class="research-header">Research</h1>
                    <p class='profile-statement'> We use simulation and theory to predict phase behavior of polymeric systems ranging from synthetic to bio macromolecules. Polymer liquid-liquid phase separation (LLPS) is critically important in formation of membrane-less organelles and design of artificial complexes with particular functions, which can be used in biotechnical and biomedical applications such as drug confinement and its targeted delivery.

                        One of our main areas of interest is to understand Tau LLPS and aggregation that is known to be critically associated with several neurodegenerative diseases. The challenges associated with characterizing the collective properties are particularly acute in Tau, where a broad range of intra- and intermolecular interactions make it elusive to link subtle residue level information to mesoscopic assemblies. We employ a combination of field-theoretic simulations (FTS), and enhanced sampling techniques such as replica exchange molecular dynamics (REMD) and indirect umbrella sampling (INDUS) to determine Tau phase behavior and understand the molecular mechanisms that lead to Tau self-assembly. Additionally, we are interested in classifying the toxicity level of Tau oligomers/fibers by characterizing the hydrophobicity strength that is presumably associated with the surface topography of the Tau complex and its chemical context.</p>
   
                    <h1 class="publication-header">Featured Publications</h1>
                    <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/pro.4101" target="_blank">
                        <strong>Liquid-liquid phase separation of Tau by self and complex coacervation</strong>
                        </a>
                        <br />
                        <p> Saeed Najafi, Yanxian Lin, Andrew P. Longhini, Xuemei Zhang, Kris T. Delaney, Kenneth S. Kosik, Glenn H. Fredrickson, Joan-Emma Shea, Songi Han <em>Prot. Sci.</em> <strong>2021,</strong> 30 (7) </p>

                        <a href="https://doi.org/10.1021/acs.chemrev.0c01122" target="_blank">
                            <strong> Amyloid Oligomers: A Joint Experimental/Computational Perspective on Alzheimer's Disease, Parkinson's Disease, Type II Diabetes and Amyotrophic Lateral Sclerosis.</strong></a>
                            <br />
                            <p> Nguyen, P. H.; Ramamoorhty, A.; Sahoo, B.; Zheng, J.; Chiricotto, M.; Straub, J. E.; Dominguez, L.; Shea, J.-E.; Dokholyan, N.; De Simone, A.; Ma, B.; Nussinov, R.; Najafi, S.; McCarty, J.; Loquet, A.; Faller, P.; Ganguly, P.; Ngo, S. T.; Li, M. S.; Hall, C.; Wang, Y.; Miller, Y.; Melchionna, S.; Habenstein, B <em>Chem Rev</em> <strong>2021,</strong> <em>121 (4), 2545-2647</em></p>
                    <h1 class="contact-header">Contact Info</h1>
                    <p> snajafi@ucsb.edu &SmallCircle; <a href='https://www.linkedin.com/in/saeed-najafi-569b8a54/' target="_blank"> LinkedIn</a></p>
                </div>
            </div>
            </div>
        </div>
    </div>
    <h3 class="role-title">Graduate Researchers</h3>

    <div id="accordion-graduate">
        <div class="card">
            <div class="card-header" id="Sheppard">
            <h5 class="mb-0 minor-title-head">
                <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#grad-collapseOne" aria-expanded="false" aria-controls="grad-collapseOne">
                Jackson Sheppard
                </button>
            </h5>
            </div>
        
            <div id="grad-collapseOne" class="collapse" aria-labelledby="headingOne" data-parent="#accordion-graduate">
            <div class="card-body">
            
                <div>
                    <p style="float: left; padding-right: 2rem; margin-bottom: 15rem"><img class="people-profile-pic" src="Images/People/jackson.jpg" alt="A picture of Jackson Sheppard"></p>
                    </div>
                    <div>
                    <h1 class="research-header">Research</h1>
                    <p class='profile-statement'> Simulations in the context of physical chemistry can generate large quantities of data warranting modern approaches in machine learning and data science to model these complex physical systems. I am interested in these computational and other physics-based approaches to model chemical and biological systems. I obtained my Bachelor of Science in Physics in 2018 from the University of California: Santa Barbara, working as an undergraduate in the Shea group under the guidance of Nathaniel Charest and Dr. Joan-Emma Shea on machine learning approaches to model the self-assembly of coarse-grained peptides. I then worked at SLAC National Accelerator Laboratory in Menlo Park, CA starting in 2018, initially under the guidance of Dr. Claudio Pellegrini to optimize the undulator profile of the Free Electron Laser at LCLS to maximize photon output power, and later as a controls engineer supporting user experiments and designing motion control systems for LCLS instruments. I then joined the Shea group as a graduate student in Chemistry in the Fall of 2021 with a focus in physical chemistry.
            
                    </p>
   
                    <h1 class="contact-header">Contact Info</h1>
                    <p> sheppard@chem.ucsb.edu &SmallCircle; <a href='https://www.linkedin.com/in/jsheppard95/' target='_blank'> LinkedIn</a></p>
                </div>
            </div>
            </div>
        </div>
        <div class="card">
            <div class="card-header" id="headingTwo">
            <h5 class="mb-0">
                <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#grad-collapseTwo" aria-expanded="false" aria-controls="grad-collapseTwo">
                Sam Lobo
                </button>
            </h5>
            </div>
            <div id="grad-collapseTwo" class="collapse" aria-labelledby="headingTwo" data-parent="#accordion-graduate">
            <div class="card-body">
                <div>
                    <p style="float: left; padding-right: 2rem; margin-bottom: 15rem"><img class="people-profile-pic" src="Images/People/sam.jpg" alt="A picture of Sam Lobo"></p>
                    </div>
                    <div>
                    <h1 class="research-header">Research</h1>
                    <p class='profile-statement'> I study Intrinsically Disordered Proteins (IDPs). Nearly 50% of all human proteins are disordered or have extensive disordered regions. I primarily study the tau protein, which misfolds in Alzheimer's disease and CTE (the concussion disease). I use various computational techniques such as molecular dynamics simulations to study fragments of the tau protein. I obtained a Bachelor of Science in Chemical Engineering from Northwestern University in 2018, working as an undergraduate researcher in the Stupp Lab studying the self-assembly of peptide amphiphiles. I was a high school physics teacher at Providence Cristo Rey High School for 2 years after undergrad. I am jointly advised by Scott Shell and Joan-Emma Shea; I joined the Shell Lab and Shea Lab in the Fall of 2020.
            
                    </p>
   
                    <h1 class="contact-header">Contact Info</h1>
                    <p> lobo@ucsb.edu &SmallCircle; <a href='https://www.linkedin.com/in/samuel-lobo-060572b0/' target='_blank'> LinkedIn</a></p>
                </div>
            </div>
            </div>
        </div>

        <div class="card">
            <div class="card-header" id="headingThree">
            <h5 class="mb-0">
                <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#grad-collapseThree" aria-expanded="false" aria-controls="grad-collapseThree">
                Somaiyeh Shokri Kalan
                </button>
            </h5>
            </div>
            <div id="grad-collapseThree" class="collapse" aria-labelledby="headingThree" data-parent="#accordion-graduate">
            <div class="card-body">
                <div>
                    <p style="float: left; padding-right: 2rem; margin-bottom: 15rem"><img class="people-profile-pic" src="Images/People/somaiyeh.jpg" style="width:150px !important;height:210px !important;" alt="A picture of Somaieyh Shokri Kalan"></p>
                    </div>
                    <div>
                    <h1 class="research-header">Research</h1>
                    <p class='profile-statement'> I received my Bachelor's and Master's degrees in Physics, and currently I am a graduate student in the Shea Group at UCSB. My interests include soft matter and biochemistry systems, with a specific focus on protein-surface adhesion through the use of molecular dynamics simulations.
            
                    </p>
   
                    <h1 class="contact-header">Contact Info</h1>
                    <p> shokrikalan@ucsb.edu </p>
                </div>
            </div>
            </div>
        </div>

    </div>

    <h3 class="role-title">Former Members</h3>

    <div id="accordion-former">
        <div class="card">
            <div class="card-header" id="Charest">
            <h5 class="mb-0 minor-title-head">
                <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#former-collapseOne" aria-expanded="false" aria-controls="former-collapseOne">
                Nathaniel Charest &SmallCircle; Ribozyme Sequence Analysis, Latent Model Development &amp; Peptide Aggregation 
                </button>
            </h5>
            </div>
        
            <div id="former-collapseOne" class="collapse" aria-labelledby="headingOne" data-parent="#accordion-former">
            <div class="card-body">
            
                <div>
                    <p style="float: left; padding-right: 2rem; margin-bottom: 15rem"><img class="people-profile-pic" src="Images/People/nate_charest.png" alt="A picture of Nathaniel Charest"></p>
                    </div>
                    <div>
                    <h1 class="research-header">Research</h1>
                    <p class='profile-statement'> My work features the use of machine learning techniques to extract insights from biopolymer primary sequences and molecular dynamics outputs. 
                        Prior collaborations have included work on detection of amyloid propensity using mass spectrometry methods and exploring methods of solvation simulation with coarse grained models.
                        I am currently pursuing applications of information theory to capture the complex interactions of genotype and phenotype within ribozymes as part of a collaboration with
                        Dr. Irene Chen's group at UCLA. We are doing this to develop better methods of characterizing how ribozymes acquire there biochemical activities and understand how to navigate the sequence spaces that define the function of these evolution-relevant biopolymers. I am always looking for new collaborations that will allow me to analyze high-quality data! </p>
   
                    <h1 class="publication-header">Featured Publications</h1>
                    <a href="https://doi.org/10.1021/acs.jpcb.0c05763" target="_blank">
                        <strong>Latent models of molecular dynamics data: Automatic order parameter generation for peptide fibrillization</strong>
                        </a>
                        <p> Charest, N.; Tro, M.; Bowers, M. T.; Shea, J.-E. <em>J. Phys. Chem. B</em> <strong>2020,</strong> <em>124 (37), 8012-8022</em> </p>

                        <a href="https://doi.org/10.1021/acs.jpcb.9b03415" target="_blank">
                            <strong>The classifying autoencoder: Gaining insight into amyloid assembly of peptides and proteins</strong>
                            </a>
                            <p> Tro, M. J.; Charest, N.; Taitz, Z.; Shea, J.-E.; Bowers, M. T. <em>J. Phys. Chem. B</em> <strong>2019,</strong> <em>123, 5256-5264. </em> </p>
                    <h1 class="contact-header">Contact Info</h1>
                    <p> ncharest@ucsb.edu &SmallCircle; <a href='https://www.linkedin.com/in/nathaniel-m-charest' target='_blank'> LinkedIn</a></p>
                </div>
            </div>
            </div>
        </div>
    </div>

<!-- Option 1: jQuery and Bootstrap Bundle (includes Popper) -->
<script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
<script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.bundle.min.js" integrity="sha384-Piv4xVNRyMGpqkS2by6br4gNJ7DXjqk09RmUpJ8jgGtD7zP9yug3goQfGII0yAns" crossorigin="anonymous"></script>
</body>
</html>